jaseci_ai_kit/jac_nlp/jac_nlp/bart_sum/bart_sum.py

```diff
@@ -6,18 +6,32 @@
 from fastapi import HTTPException
 import requests
 from bs4 import BeautifulSoup
-
+from jaseci.utils.utils import model_base_path
+import os
 
 device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+BART_SUM_ROOT = model_base_path("jac_nlp/bart_sum")
 
 
 @jaseci_action(act_group=["bart_sum"], allow_remote=True)
 def setup(
     tokenizer: str = "facebook/bart-large-cnn", model: str = "facebook/bart-large-cnn"
 ):
     global bart_tokenizer, bart_model
-    bart_tokenizer = BartTokenizer.from_pretrained(tokenizer)
-    bart_model = BartForConditionalGeneration.from_pretrained(model).to(device)
+    os.makedirs(BART_SUM_ROOT, exist_ok=True)
```

Comment:
Can you make sure you review any actual code logic changes in any of the jac_nlp modules? after resolving the conflicts of course. 

Let's make sure this doesn't break any of the module. The tests should help here too but a manual review would be nice.